src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3690,13 +3711,14 @@ _test_sample_versioned_api_example_4 (void)
     * client = mongoc_client_new (uri_sharded);
     */
 
-   client = get_client ();
+   client = get_client_for_version_api_example ();
 
    mongoc_server_api_version_from_string ("1", &server_api_version);
    server_api = mongoc_server_api_new (server_api_version);
    mongoc_server_api_deprecation_errors (server_api, true);
 
-   mongoc_client_set_server_api (client, server_api, &error);
+   ok = mongoc_client_set_server_api (client, server_api, &error);
```

Comment:
nit: I'm ok with "ok", but it might be okay-er to just check the call with BSON_ASSERT directly, if this is the only place we're using it. But, really, I'm ok either way!

Reply:
Done, here and in the other changes!